lib/perl/Genome/Model/ReferenceAlignment/Command/RefCovMetrics.pm

```diff
@@ -0,0 +1,124 @@
+package Genome::Model::ReferenceAlignment::Command::RefCovMetrics;
+
+use strict;
+use warnings 'FATAL';
+
+use Genome;
+
+require List::MoreUtils;
+
+class Genome::Model::ReferenceAlignment::Command::RefCovMetrics {
+    is => 'Command::V2',
+    has_input => {
+        models => {
+            is => 'Genome::Model::ReferenceAlignment',
```

Comment:
Should we make this operate on any `Genome::Model`, go through `$build->results` to find ref-cov results (of which there may be two for `SomaticValidation`, and call the accessors on the results directly?